src/app/code/community/Omise/Gateway/Model/OmiseCharge.php

```diff
@@ -17,7 +19,9 @@ public function createOmiseCharge($params)
 
     /**
      * Capture a charge that retrieve from charge id
-     * @param array $params
+     *
+     * @param  array $params
```

Comment:
Updated in https://github.com/omise/omise-magento/pull/40/commits/ab6bde3a3a5308ed1e636e81546348dc2472d8ba 👍